src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.scss

```diff
@@ -67,6 +69,17 @@
         line-height: 24px;
     }
 
+    &__issues {
```

Comment:
Can you call this class somehow differently? They are not the issues itself, but their count